figtree_test.go

```diff
@@ -36,6 +36,16 @@ func newFigTreeFromEnv() *FigTree {
 	return NewFigTree(os.Getenv("HOME"), cwd, "FIGTREE")
 }
 
+func applyEnvSet(envSet map[string]*string) {
```

Comment:
I think we should make this exported so that users can deal with Figtree easier to get back to the current behavior.